packages/lambda-tiler/src/routes/__tests__/xyz.test.ts

```diff
@@ -110,6 +111,7 @@ o.spec('/v1/tiles', () => {
     o(res.header('content-type')).equals('image/png');
     o(res.header('etag')).notEquals(undefined);
     o(res.header('cache-control')).equals('public, max-age=604800, stale-while-revalidate=86400');
+    o(res.header('content-disposition')).equals('inline; filename="🦄 🌈_0_0_0.png"');
```

Comment:
TODO: I am not sure we are allowed to just jam UTF8 characters into filenames, maybe we need to encode them first.